src/lib.rs

```diff
@@ -167,7 +167,10 @@ mod builder;
 pub use builder::{KnownHosts, SessionBuilder};
 
 mod command;
-pub use command::Command;
+pub use command::{Command, OverSsh};
+
+mod escape;
+pub use escape::escape;
```

Comment:
No, let's not export this.
It should not be part of the public API.

Reply:
Actually, I'd like this to be a separate crate, but that is out of the scope of this PR.

Let's just make this crate-internal API.